webapp/src/cljc/lipas/roles.cljc

```diff
@@ -0,0 +1,94 @@
+(ns lipas.roles)
+
+(def privileges
+  {:create {:doc "Oikeus lisätä liikuntapaikkoja"}
```

Comment:
Please nest under localization key `{:doc {:fi "Oikeus lisätä..."}}`. As a "publicly funded" system, Lipas needs to support also English and Swedish. No need to do the translations, since they can be done later by a translator.

Note, for historical reasons we use use `:se` instead of `:sv` for Swedish localization.

Reply:
I'm not yet sure if this is the user-readable name and how to handle localization keys. I now consider this a developer docstring.